doc/content/lorawan/rssi-and-snr/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "RSSI and SNR"
+section: Additional Information
+description: ""
+weight:
+---
+
+In wireless communication, a receiver needs a good signal strength and a signal-to-noise ratio to separate the original information or signal from the modulated carrier.
+
+## RSSI
+
+**RSSI (Received Signal Strength Indicator)** is a measurement that helps you to determine if the receiver is receiving enough signal (signal strength) to get a good wireless connection from the transmitter. Since LoRaWAN supports bi-directional communication, RSSI is an important measurement for both gateways and end devices. RSSI is measured in dBm and it is a negative value. An RSSI value that is closer to zero usually means a better signal strength.
+
+Apart from the output power of the transmitter, the following factors mainly influence the RSSI. 
+
+- Path loss
+- Antenna gain
+- Cable/connector loss
+
+## SNR
+
+**Signal-to-Noise Ratio**, often written as SNR or S/N, is the difference between the power of the received signal and the noise floor. SNR can be used to determine the quality of the received signal.
+
+SNR is calculated using the following formula and is typically expressed in decibels (dB).
+
+**SNR (dB) = received signal strength - noise floor**
```

Comment:
I would say "... = P_received_signal (dB) - P_noise (dB)" or something similar to express the fact that these also have to be in dB